config/backpack/base.php

```diff
@@ -43,8 +43,12 @@
         // 'packages/noty/noty.css',
 
         // Load the fonts separately (so that you can replace them at will):
-        'packages/source-sans-pro/source-sans-pro.css',
-        'packages/line-awesome/css/line-awesome.min.css',
+        ['https://fonts.googleapis.com/css2?family=Source+Sans+Pro:ital,wght@0,400;0,600;0,700;1,400&display=swap', true, [], 'style'],
```

Comment:
This Google Font CSS file got downloaded for me, but in the end, it didn't get used by the interface. See the difference between BEFORE (`main` branch) and AFTER (`bassets` branch):

<img width="1512" alt="CleanShot 2023-01-03 at 16 59 29@2x" src="https://user-images.githubusercontent.com/1032474/210382831-d14854a3-9aa9-4839-be0b-d37d0998d54c.png">

I remember @promatik told me we have this issue, and we talked about possible ways to solve it. So yeah, we should find a solution for this ASAP.